packages/compass-indexes/src/components/indexes-table/indexes-table.tsx

```diff
@@ -58,20 +59,35 @@ const nameFieldStyles = css({
   paddingBottom: spacing[2],
 });
 
+const tableStyles = css({
+  thead: {
+    position: 'sticky',
+    top: 0,
+    background: uiColors.white,
+    zIndex: 1,
+  },
+});
+
 type IndexesTableProps = {
-  darkMode?: boolean;
   indexes: IndexDefinition[];
   canDeleteIndex: boolean;
   onSortTable: (column: SortColumn, direction: SortDirection) => void;
   onDeleteIndex: (name: string) => void;
 };
 
+const getContainerHeight = (): string => {
+  // 300px for the compass tabs, collection info etc
+  return `${window.innerHeight - 300}px`;
```

Comment:
Is it some min form high or from where we know that it should be 300?

Reply:
its the calculated height from various component on the compass screen like mongosh shell at the bottom, top nav tabs, collection name/info row and then content tabs. This approximately adds upto to 300px

Reply:
What happens if this value gets changed in another pr (for example we shrink the header or remove the tabs)?

Reply:
the way compass sets up currently, this component is mounted first when the user is not on indexes screen yet.

as a result i had to put some initial height (window.innerHeight - 320px) which works well with the current compass. after the component renders, the height is calculated from the ref container.

Reply:
Is it possible maybe to subtract then not 320px but something like `window.innerHeight - window.innerHeight * 0.5`? like some percentage of height (find a proper value here that past).